dll/cpl/desk/settings.c

```diff
@@ -377,6 +380,8 @@ SettingsOnInitDialog(IN HWND hwndDlg)
 
         OnDisplayDeviceChanged(hwndDlg, pData, pData->DisplayDeviceList);
 
+        ShowWindow(GetDlgItem(hwndDlg, IDC_RESOLUTION_PREVIEW), SW_HIDE);
```

Comment:
Are the controls correctly shown again *if* one adds or removes at runtime a monitor while keeping the desk.cpl tab opened? (I don't know whether Windows updates that at runtime as well :thinking:)

Reply:
I have checked Windows desk.cpl, it does not handle resolution changes in runtime (i.e. when the corresponding page already has initialzied), it probably does not handle monitor hotplugging as well (I don't know any emulator supporting that feature).

Reply:
Note:
1. on windows, the equivalent functionality touched in this PR is in themeui.dll (loaded and displayed by desk.cpl afterwards)
2. windows doesn't update the page even if you go to another page, and then go back?

Reply:
1. Yes, this is correct. This is also why you need both desk.cpl/themeui.dll from XP when testing in ReactOS.
2. Yes, it doesn't react to the changes even when switching pages.